src/actions/photosActions.ts

```diff
@@ -596,6 +596,23 @@ export function generatePhotoIm2txtCaption(image_hash: string) {
   };
 }
 
+
+export function savePhotoCaption(image_hash: string, caption?: string | undefined) {
+  return function (dispatch: Dispatch<any>) {
+    dispatch({ type: "SAVE_PHOTO_CAPTION" });
```

Comment:
SAVE_PHOTO_CAPTION does nothing, because there redux event, which would consume that